### PR TITLE
SRCH-4122 Update ASIS README about using docker

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Updated ReadMe for using ASIS within docker.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,21 +26,32 @@ Use [rvm](https://rvm.io/) to install the version of Ruby specified in `.ruby-ve
...
 
-We use bundler to manage gems. You can install bundler and other required gems like this:
+Docker can be used to: 1) run just the required services (MySQL, Elasticsearch, etc.) while [running the asis application in your local machine](https://github.com/GSA/asis#developmentusage), and/or 2) run the entire `asis` application in a Docker container.  Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for the services.
 
-    gem install bundler
-    bundle install
+When running in a Docker container (option 2 above), the `asis` application is configured to run on port [3300](http://localhost:3300/). Required dependencies - ([Ruby](https://github.com/GSA/asis#ruby), and [Gems](https://github.com/GSA/asis#ruby)) - are installed using Docker. However, other data or configuration may need to be setup manually, which can be done in the running container using `bash`.
 
-### Services
+Using bash to perform any operations on asis application running in Docker container, below command needs to be run in `search-services`.
```
Comment (MyNameIsMissing) [inline README.md:35]: Can we reword this?  Phrasing seems a little awkward and I'm not sure what it is referring to

**Review by MyNameIsMissing — CHANGES_REQUESTED** [review]
Left a request for rephrasing one line.  The rest looks good.

File: README.md (hunk 1 above)
Comment (krbhavith) [reply README.md]: @MyNameIsMissing please review the updated phrase.

**Review by MyNameIsMissing — APPROVED** [review]
Thanks.  That makes sense to me now.

LGTM

**Pull request merged**